### DISCUSSION TITLE
Can wg-easy + Pi-hole work with host server IP for DNS?

### DISCUSSION THREAD

**Discussion by zero-thermo** [wg-easy/wg-easy#531] [Q&A]
I have wg-easy and Pi-hole (AdGuard Home) working thanks to the [wiki guide](https://github.com/wg-easy/wg-easy/wiki/Using-WireGuard-Easy-with-Pi-Hole). The sample compose file uses internal Docker IPs (10.8.1.0/24) for DNS. Both services are installed on the same host machine.   

When I try switching to the host machine IP (192.168.1.2) for DNS, I no longer get DNS resolution. However, I can still access other Docker service on the same host by using IP:PORT.  

If I am able to access the host IP for other services while connected to wg-easy, why can't I also use the host IP to connect to my AdGuard DNS?  

Thanks for working on this great service.

**Comment by DigneZzZ** [comment]
See my  repo and you can ask a question on my forum: openode.ru

**Comment by pheiduck** [comment]
`- WG_DEFAULT_DNS=192.168.1.2` should be working if you use the IP where Pi-hole is running on.
